R/as_flextable.R

```diff
@@ -503,7 +503,7 @@ tt_to_flextable <- function(tt,
 #' @describeIn tt_to_flextable Main theme function for [export_as_docx()].
 #'
 #' @param font (`string`)\cr font. Defaults to `"Arial"`. If the font given is not available, the `flextable` default
-#'   is used instead. For options, consult the family column from [systemfonts::system_fonts()].
+#'   is used instead. For options, consult the family column from `systemfonts::system_fonts()`.
```

Comment:
Solves #36. It is only a suggestion, I do not think we need to make the connection evident as we do not depend on this